config/environments/production.rb

```diff
@@ -91,4 +91,6 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  routes.default_url_options = { host: 'enbw.herokuapp.com', port: 80, protocol: 'http' }
```

Comment:
These get redirected to `https` by default on Heroku.

Reply:
Good to know. Probably need to get a subdomain of their actual domain redirected over here plus some Let's Encrypt action, but let's start with getting it back on the TVs.